simapp/app.go

```diff
@@ -76,21 +76,23 @@ func MakeCodec() *codec.Codec {
 	return cdc
 }
 
-// Extended ABCI application
+// App extends an ABCI application, but with most of its parameters exported.
+// They are exported for convenience in creating helper functions, as object
+// capabilities aren't needed for testing.
 type SimApp struct {
 	*bam.BaseApp
-	cdc *codec.Codec
```

Comment:
We should rather keep this private and provide a `Codec() *codec.Codec` method on `SimApp`.